S21/indraneelL/wordConvert.py

```diff
@@ -0,0 +1,29 @@
+import locale
+import re
+
+## read the file and iterate through every line
+class WordConvert:
+    def readTestFile(self):
+        textFile = open("tests.tsv", "r")
+        while textFile:
+            line = textFile.readline()
+            if line == "" or line == '\n':
+                break
+            word, lang, _  = line.split('\t')
+            self.methodToProcessText(word,lang)
+        textFile.close()
+
+    def makeWordsLower(self,word):
+        print(word.lower())
+
+    def methodToProcessText(self, text, lang):
+        # using regex check for the language and do functionalities
+        word = text
+        if re.findall("\A[n,t][AEIOUÁÉÍÓÚ]",text) and re.match("^ga.*",lang):
+            word = re.sub(r'(^[n,t])([AEIOUÁÉÍÓÚ])', r'\1-\2',text)
+        elif re.match("^tr.*", lang):
+            word = text.replace("I","ı")
+        self.makeWordsLower(word.lower())
+
+wordConvert = WordConvert()
+wordConvert.readTestFile()
```

Comment:
I believe there's no need to need to write  a code for the exception of "greek" language. Could you please add a comment explain why.

Reply:
Yes, Since Python 3.0, all strings are stored as Unicode in an instance and adding functionality for Σ and σ does the same as default, so i am getting the desired output of the column 3 and so, i have removed the functionality for greek and other languages.